docs/guide/scenes/crazyflie/crazyflie.x3d

```diff
@@ -0,0 +1,17 @@
+<Transform id='n9' docUrl='https://cyberbotics.com/doc/reference/robot' name='robot' controller="web_component_studio_supervisor" window="&lt;generic&gt;" type='robot'></Transform>
+<Transform id='n10' docUrl='https://cyberbotics.com/doc/guide/crazyflie' translation='0 0 0.05' name='crazyflie' controller="crazyflie" type='robot'><Transform id='n11' docUrl='https://cyberbotics.com/doc/reference/transform' translation='0 0 -0.015'><CadShape id='n13' docUrl='https://cyberbotics.com/doc/reference/cadshape' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/cf2_assembly.dae"'></CadShape></Transform><Transform id='n14' docUrl='https://cyberbotics.com/doc/reference/transform' translation='0 0 0.02'></Transform><Group type='propeller'><Transform id='n25' docUrl='https://cyberbotics.com/doc/reference/solid' translation='0.031 -0.031 0.008' name='solid' type='solid'><Shape id='n26' castShadows='true'><PBRAppearance id='n27' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' baseColor='0 0 0' metalness='0.3'></PBRAppearance><Mesh id='n28' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/ccw_prop.dae"'></Mesh></Shape></Transform></Group><Group type='propeller'><Transform id='n31' docUrl='https://cyberbotics.com/doc/reference/solid' translation='-0.031 -0.031 0.008' name='solid' type='solid'><Shape id='n32' castShadows='true'><PBRAppearance id='n33' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' USE='n27'/><Mesh id='n34' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/cw_prop.dae"'></Mesh></Shape></Transform></Group><Group type='propeller'><Transform id='n37' docUrl='https://cyberbotics.com/doc/reference/solid' translation='-0.031 0.031 0.008' name='solid' type='solid'><Shape id='n38' castShadows='true'><PBRAppearance id='n39' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' USE='n27'/><Mesh id='n40' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/ccw_prop.dae"'></Mesh></Shape></Transform></Group><Group type='propeller'><Transform id='n43' docUrl='https://cyberbotics.com/doc/reference/solid' translation='0.031 0.031 0.007' name='solid' type='solid'><Shape id='n44' castShadows='true'><PBRAppearance id='n45' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' USE='n27'/><Mesh id='n46' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/cw_prop.dae"'></Mesh></Shape></Transform></Group><Cylinder id='n47' docUrl='https://cyberbotics.com/doc/reference/cylinder' height='0.03' radius='0.05' subdivision='36'></Cylinder></Transform>
```

Comment:
This is one of the generated files right? it seems it still has the old value

```suggestion
<Transform id='n10' docUrl='https://cyberbotics.com/doc/guide/crazyflie' translation='0 0 0.015' name='crazyflie' controller="crazyflie" type='robot'><Transform id='n11' docUrl='https://cyberbotics.com/doc/reference/transform' translation='0 0 -0.015'><CadShape id='n13' docUrl='https://cyberbotics.com/doc/reference/cadshape' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/cf2_assembly.dae"'></CadShape></Transform><Transform id='n14' docUrl='https://cyberbotics.com/doc/reference/transform' translation='0 0 0.02'></Transform><Group type='propeller'><Transform id='n25' docUrl='https://cyberbotics.com/doc/reference/solid' translation='0.031 -0.031 0.008' name='solid' type='solid'><Shape id='n26' castShadows='true'><PBRAppearance id='n27' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' baseColor='0 0 0' metalness='0.3'></PBRAppearance><Mesh id='n28' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/ccw_prop.dae"'></Mesh></Shape></Transform></Group><Group type='propeller'><Transform id='n31' docUrl='https://cyberbotics.com/doc/reference/solid' translation='-0.031 -0.031 0.008' name='solid' type='solid'><Shape id='n32' castShadows='true'><PBRAppearance id='n33' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' USE='n27'/><Mesh id='n34' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/cw_prop.dae"'></Mesh></Shape></Transform></Group><Group type='propeller'><Transform id='n37' docUrl='https://cyberbotics.com/doc/reference/solid' translation='-0.031 0.031 0.008' name='solid' type='solid'><Shape id='n38' castShadows='true'><PBRAppearance id='n39' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' USE='n27'/><Mesh id='n40' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/ccw_prop.dae"'></Mesh></Shape></Transform></Group><Group type='propeller'><Transform id='n43' docUrl='https://cyberbotics.com/doc/reference/solid' translation='0.031 0.031 0.007' name='solid' type='solid'><Shape id='n44' castShadows='true'><PBRAppearance id='n45' docUrl='https://cyberbotics.com/doc/reference/pbrappearance' USE='n27'/><Mesh id='n46' docUrl='https://cyberbotics.com/doc/reference/mesh' url='"webots://projects/robots/bitcraze/crazyflie/protos/meshes/cw_prop.dae"'></Mesh></Shape></Transform></Group><Cylinder id='n47' docUrl='https://cyberbotics.com/doc/reference/cylinder' height='0.03' radius='0.05' subdivision='36'></Cylinder></Transform>
```

Reply:
I mean in the z-axis: translation='0 0 0.05'  should be 0 0 0.015.

Reply:
It depends, you want the default translation to be `0 0 0.015` when the PROTO is inserted in the world as well (in this case it requires changing the PROTO) ? Or just have a non-default pose just for the web component (in which case the json needs to be changed)?

If the PROTO needs to be modified, then the documentation page need to be updated accordingly (as it's generated from the PROTO)

Reply:
Indeed, changing the json file for the web component viewer won't change the default-pose of the proto (here 0 0 0.015).

I have modified the height to the default pose, but I personally think it looks better floating in the web component viewer (see comparison below)

![0 05](https://user-images.githubusercontent.com/100125668/176198550-672da5f0-c2c7-4af8-b948-ba1d3d4b7e61.png)
![0 015](https://user-images.githubusercontent.com/100125668/176198562-b44048c9-83a9-4247-bbea-00c42dbf49d9.png)

let me know!

Reply:
yes this looks good!